docs/tutorials/go/build-an-ecommerce-app/index.md

```diff
@@ -13,14 +13,690 @@ last_update:
 
 ![Temporal Go SDK](/img/sdk_banners/banner_go.png)
 
-import { OutdatedNotice } from '@site/src/components'
+## Introduction
 
-<OutdatedNotice />
+In this tutorial, you'll implement a web shopping cart using Temporal Workflows and Signals. 
+The example patterns here use the [Mailgun](https://www.mailgun.com/) and [Stripe](https://stripe.com/) APIs, so you will need a developer account on both platforms to follow along.
 
-In this four-part series, you'll explore how you can use Temporal to develop an eCommerce application with Temporal Workflows. You'll write the basic application, send reminder emails, write tests, and develop a REST API.
+### Prerequisites
 
-* [Build an eCommerce App With Temporal and Go, Part 1: Getting Started](./part1.md)
-* [Build an eCommerce App With Temporal and Go, Part 2: Reminder Emails](./part2.md)
-* [Build an eCommerce App With Temporal and Go, Part 3: Testing](./part3.md)
-* [Build an eCommerce App With Temporal and Go, Part 4: REST API](./part4.md)
+You can build the project by following this tutorial, or just grab the ready-to-go source from its [GitHub repo](https://github.com/temporalio/temporal-ecommerce).
+This repository has the full source code and can serve as a guide or a fallback if you encounter issues when working through this tutorial.
+If you want to play first and explore later, you can come back and read through the how-to and background.
 
+If you're new to Temporal, follow ["Set up a local development environment"](/getting_started/go/dev_environment/) so you're ready to build Temporal applications with Go.
+Ensure a local Temporal Service is running (e.g. with [the Temporal CLI](https://github.com/temporalio/cli)) and that you can access the Temporal Web UI from port `8233`.
```

Comment:
```suggestion
Ensure a local Temporal Service is running (e.g. with [the Temporal CLI](https://github.com/temporalio/cli) and that you can access the Temporal Web UI from port `8233`.
```

Reply:
Nope, that parenthesis actually matches the one from the e.g :)

Reply:
Oops! My bad !